### DISCUSSION TITLE
What is the release schedule for super-linter, related to bugfix in underlying linter

### DISCUSSION THREAD

**Discussion by mikelax** [super-linter/super-linter#5922] [Q&A] [answered]
Hello, I am not sure where to ask there, but I am wondering what is the normal release cycle, or how would someone know when a given change / dependency update will be released? 

The reason I am asking is that I see [this bug was resolved in checkov](https://github.com/bridgecrewio/checkov/issues/6172) about three weeks ago. The updated version of checkov has already been updated in the super-linter repo, it just needs to be published as a 6.x release.

**Comment by ferrarimarco** [comment, marked as answer]
Hi @mikelax ! There's currently no fixed release schedule. This is something we should address in a more structured way. Can you please open an issue about this?

Semi-related to this: we're experimenting to include patch linter updates in super-linter patch updates, while minor and major linter updates in super-linter minor updates. You might see this from the milestones.

Hope this helps!

**Comment by mikelax** [comment]
Thanks @ferrarimarco for the info. I will open an issue specific about the release process. I was also going to open an issue asking if there would be a way as a user to specify a version for a given linter, in this case `checkov`.
